docs/building_the_project_with_dpcpp.rst

```diff
@@ -408,6 +423,21 @@ set, the backend libraries to enable the use of BLAS, LAPACK and RNG with MKLGPU
 and MKLCPU would also be enabled. The build of examples is disabled. Since
 functional testing was not disabled, tests would be built.
 
+Build oneMKL for the BLAS domain on a generic SYCL device:
+
+.. code-block:: bash
+
+  cmake $ONEMKL_DIR \ 
+      -DCMAKE_CXX_COMPILER=clang++ \
+      -DCMAKE_C_COMPILER=clang \ 
+      -DENABLE_MKLCPU_BACKEND=False \ 
+      -DENABLE_MKLGPU_BACKEND=False \
+      -DENABLE_PORTBLAS_BACKEND=True
+
+Note that this is not a supported configuration. This builds oneMKL Interfaces
```

Comment:
Does it make sense to mention that we do not test it, but it's supported by portBLAS backend? To me "not supported" means we don't have backend for that, but we have, right?
```suggestion
Note that this is not a tested configuration. This builds oneMKL Interfaces
```